source/_integrations/honeywell.markdown

```diff
@@ -16,16 +16,42 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-The Honeywell integration integrates Home Assistant with _US-based_ [Honeywell Total Connect Comfort (TCC)](https://mytotalconnectcomfort.com/portal/) climate systems.
+The `Honeywell` integration integrates Home Assistant with _US-based_ [Honeywell Total Connect Comfort (TCC)](https://mytotalconnectcomfort.com/portal/) climate systems.
 
-It uses the [somecomfort](https://github.com/kk7ds/somecomfort) client library. It does not support the home security functionality of TCC.
+It uses the [aiosomecomfort](https://github.com/mkmer/AIOSomecomfort) client library. It does not support the home security functionality of TCC.
```

Comment:
This is an internal implementation detail, which isn't really a concern for the end-user. I suggest to remove it.

```suggestion
It does not support the home security functionality of TCC.
```